test/api/v3/integration/tasks/GET-tasks_user.test.js

```diff
@@ -0,0 +1,33 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
```

Comment:
This will need to change to ../../../../helpers/api-v3-integration.helper'; Will do wo when I work on it.